test/Microsoft.AspNet.Mvc.TagHelpers.Test/ImageTagHelperTest.cs

```diff
@@ -0,0 +1,265 @@
+            Assert.Equal(5, output.Attributes.Count);
+            
+            for(int i=0; i < expectedOutput.Attributes.Count; i++)
+            {
+                var expectedAtribute = expectedOutput.Attributes[i];
+                var actualAttribute = output.Attributes[i];
+                Assert.Equal(expectedAtribute.Name, actualAttribute.Name);
+                Assert.Equal(expectedAtribute.Value.ToString(), actualAttribute.Value.ToString());
+            }
+        }
+
+        [Fact]
+        public void RendersImageTag_AddsFileVersion()
+        {
+            // Arrange
+            var context = MakeTagHelperContext(
+                attributes: new TagHelperAttributeList
+                {
+                    { "alt", new HtmlString("Alt image text") },
+                    { "src", "/images/test-image.png" },
+                    { "asp-file-version", "true" }
+                });
+            var output = MakeImageTagHelperOutput(attributes: new TagHelperAttributeList
+            {
+                { "alt", new HtmlString("Alt image text") },
+            });
+            var hostingEnvironment = MakeHostingEnvironment();
+            var viewContext = MakeViewContext();
+            var helper = new ImageTagHelper
+            {
+                HostingEnvironment = hostingEnvironment,
+                ViewContext = viewContext,
+                Src = "/images/test-image.png",
+                FileVersion = true,
+                Cache = MakeCache(),
+            };
+
+            // Act
+            helper.Process(context, output);
+
+            // Assert
+            Assert.True(output.Content.IsEmpty);
+            Assert.Equal("img", output.TagName);
+            Assert.Equal(2, output.Attributes.Count);
+            TagHelperAttribute srcAttribute = Assert.Single(output.Attributes, attr => attr.Name.Equals("src"));
+            Assert.Equal("/images/test-image.png?v=f4OxZX_x_FO5LcGBSKHWXfwtSx-j1ncoSt3SABJtkGk", srcAttribute.Value);
+        }
+
+        [Fact]
+        public void RendersImageTag_AddsFileVersion_WithRequestPathBase()
+        {
+            // Arrange
+            var context = MakeTagHelperContext(
+                attributes: new TagHelperAttributeList
+                {
+                    { "alt", new HtmlString("alt text") },
+                    { "src", "/bar/images/image.jpg" },
+                    { "asp-file-version", "true" },
+                });
+            var output = MakeImageTagHelperOutput(attributes: new TagHelperAttributeList
+            {
+                { "alt", new HtmlString("alt text") },
+            });
+            var hostingEnvironment = MakeHostingEnvironment();
+            var viewContext = MakeViewContext("/bar");
+            var helper = new ImageTagHelper
+            {
+                HostingEnvironment = hostingEnvironment,
+                ViewContext = viewContext,
+                Src = "/bar/images/image.jpg",
+                FileVersion = true,
```

Comment:
Need a test with `FileVersion == false` to confirm `CopyHtmlAttribute()` is called correctly.